en/guide/index.md

```diff
@@ -22,57 +22,57 @@ As a framework, Nuxt.js comes with a lot of features to help you in your develop
 
 ## How it Works
 
-![Vue with Webpack and Babel](https://i.imgur.com/avEUftE.png)
+![Vue with webpack and Babel](https://i.imgur.com/avEUftE.png)
 
 Nuxt.js includes the following to create a rich web application development:
 
 - [Vue 2](https://github.com/vuejs/vue)
-- [Vue-Router](https://github.com/vuejs/vue-router)
+- [Vue Router](https://github.com/vuejs/vue-router)
```

Comment:
As in official Vue documentation the name of Product is « Vue Router » and the name of package is « vue-router ». Vue-Router is not an official term. We suggiest use the product name as « Vue 2 »
— source : https://vuejs.org/v2/guide/migration-vue-router.html